app/assets/stylesheets/screen.scss

```diff
@@ -626,6 +623,11 @@ div#keycut-help {
 .sign-in .checkbox {
   margin-left: 50px;
 }
+.devise_links {
```

Comment:
We should use `devise-links` here instead of `devise_links` to follow the standard of the other class names.